server.js

```diff
@@ -18,8 +19,66 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
+var securityHeaders = {
+  directives: {
+    defaultSrc: [
+      `\'self\'`
+    ],
+    scriptSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`
+    ],
+    fontSrc: [
+      `\'self\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `data:`,
+      `*`
+    ],
+    connectSrc: [
+      `\'self\'`,
+      env.PULSE_API || `https://network-pulse-api-staging.herokuapp.com/`,
+      `*`
+    ]
+  },
+  reportOnly: false,
+  browserSniff: false
+};
+
+app.use(helmet.contentSecurityPolicy(securityHeaders));
```

Comment:
All of the project thumbnails we are showing (e.g., https://pulse-react.herokuapp.com/featured) are hotlink images. That's why I had to add ``*`` to `imgSrc`. 

We also have anchors that link out to each of the project's homepage so ``*`` is there in `connectSrc`. (though after reading docs on MDN I'm still not super sure about what I was supposed to put for `connectSrc`). We make XHR to Pulse API and have external links to places that we can't predict. I guess in that case it makes sense that I `*` there???